Source/WebKit/Shared/Cocoa/ArgumentCodersCocoa.mm

```diff
@@ -276,6 +283,10 @@ NSType typeFromObject(id object)
     if ([object isKindOfClass:[WebCore::CocoaColor class]])
         return NSType::Color;
 #if ENABLE(DATA_DETECTION)
+#if PLATFORM(MAC)
+    if (PAL::isDataDetectorsCoreFrameworkAvailable() && [object isKindOfClass:[PAL::getWKDDActionContextClass() class]])
```

Comment:
Here and below, I'm surprised that the class selector call is necessary.

Reply:
I'm not sure I follow the comment.
The purpose of the function is to match classes to enum values.